cluster/corosync.sls

```diff
@@ -5,9 +5,9 @@ customize_corosync:
     - name: /etc/corosync/corosync.conf
     - data: {{ cluster.corosync|json }}
 
-corosync_service:
-  service.running:
-    - name: corosync
+reload_corosync_configuration:
+  cmd.run:
+    - name: corosync-cfgtool -R
```

Comment:
@arbulu89 According this https://github.com/corosync/corosync/wiki/Config-file-values
Not all configuration item will be reloaded via `corosync-cfgtool -R`

Reply:
Thanks @liangxin1300 ,
Considering this, I will need to do more research. The complete restart of the daemon is needed in some cases, and this affects the formula usage, as a sudden restart of the cluster might affect the other node joining process

Reply:
@liangxin1300 Unfortunately some of the variables we update (`secauth` for example) are not covered in `corosync-cfgtool`, so the restart looks needed.
The restart has the big side-effect that stops the cluster completely, so if there is some node joining, this operation will fails.
This makes things more difficult.
In the past, we had a specific `lock` for the formula, which we removed when the lock was implemented in `crmsh`, but this was solving the issue, as we could atomize operations outside of `crmsh`. Now I need to see if there is some other alternative or we need to go back to the lock mechanism, which I don't want.

Reply:
@arbulu89 If there is some node joining, on init node, there should exists `/run/.crmsh_lock_directory`
How about here wait for `/run/.crmsh_lock_directory` removed, then execute restarting corosync.service?

Reply:
Yes, good idea. This can be an option. I will need to do some testing because there might be some gap between the joining process creates this file and the corosync service is restarted. To assure everything I would need to create this lock file even in this formula before restarting the corosync service as final resort.
I will keep you updated @liangxin1300